.gitignore

```diff
@@ -0,0 +1 @@
+./node_modules
```

Comment:
this should just be node_modules - all of the node_modules folders and files are now part of the repository, which means I can't comment on all of the files